src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -0,0 +1,499 @@
+
+#### `hostNetwork` and `privileged` [#hostnetwork-privileged]
+
+Most users and Kubernetes distributions configure the control plane metrics endpoints to listen only in the loopback interface, i.e. `localhost`. For this reason, the control plane component is deployed with `hostNetwork: true` by default when `privileged` is set to `true` (the default).
+
+To honor the master `privileged` setting, `hostNetwork` will be set to `false` if `privileged` is set to `false`. This will cause control plane scraping to fail in most environments, which will result in missing metrics or the `nrk8s-controlplane` pods getting stuck into a `CrashLoopBackoff` state. This is a limitation of Kubernetes itself, as control plane cannot be monitored without `hostNetwork` unless components are manually configured to do so.
+
+However, it is a common setting to deploy the integration in unprivileged mode (`privileged: false`), but still consider acceptable running the control plane pods with `hostNetwork`. This can be achieving by setting `controlPlane.unprivilegedHostNetwork` to `true`: This will tell the chart to deploy the control plane component with `hostNetwork: true`, despite the value of the higher-level `privileged` flag.
+
+If running pods with `hostNetwork` is not acceptable whatsoever, due to cluster or other policies, control plane monitoring is not possible and should be disabled by setting `controlPlane.enabled` to `false`.
+
+#### Custom autodiscovery [#autodiscovery-default]
+
+Selectors used for autodiscovery are completely exposed as configuration entries in the `values.yaml` file, which means they can be tweaked or replaced to fit almost any environment where the control plane is run as a part of the cluster.
+
+An autodiscovery section looks like the following:
+
+```yaml
+autodiscover:
+  - selector: "tier=control-plane,component=etcd"
+    namespace: kube-system
+    # Set to true to consider only pods sharing the node with the scraper pod.
+    # This should be set to `true` if Kind is Daemonset, `false` otherwise.
+    matchNode: true
+    # Try to reach etcd using the following endpoints.
+    endpoints:
+      - url: https://localhost:4001
+        insecureSkipVerify: true
+        auth:
+          type: bearer
+      - url: http://localhost:2381
+  - selector: "k8s-app=etcd-manager-main"
+    namespace: kube-system
+    matchNode: true
+    endpoints:
+      - url: https://localhost:4001
+        insecureSkipVerify: true
+        auth:
+          type: bearer
+```
+
+The `autodiscover` section contains a list of autodiscovery entries. Each entry has:
+* `selector`: A string-encoded label selector that will be used to look for pods.
+* `matchNode`: If set to true, it will additionally limit discovery to pods running in the same node as the particular instance of the DaemonSet performing discovery.
+* `endpoints`: A list of endpoints to try if a pod is found for the specified selector.
```

Comment:
Does it make sense to replicate the content here or can we point them to helm values?

Reply:
or it is here in case they do not use helm?

Reply:
In this case I chose to put it here as it is close to the explanations, so users don't have to open a different link to see what I'm talking about.

Reply:
resolved